pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -265,6 +266,9 @@ def setUpClass(cls):
             cls.sync_repo,
         )
         cls.task_bodies = tuple(poll_spawned_tasks(cls.cfg, report))
+        cls.repos_iter = tuple((
+            get_repo(cls.cfg, attrs['_href']) for attrs in cls.attrs_iter
+        ))
```

Comment:
> Can we just not use that attribute name? It's confusing, and doesn't seem to be helping. What if the last line is:
>
> `self.repo = get(cls.cfg, repo_href)`

We could. But at that point, we do have duplication between the `attrs_iter` and `repo` variables. They both describe the same thing.

Another option is to read the repository in the test case itself. Reading doesn't mutate anything server-side, so this shouldn't affect any tests:

```python
    def test_unit_count_on_repo(self):
        """Verify that the sync added the correct number of units to the repo.

        Looks at the content counts on the repo.

        This also verifies that the counts themselves are getting set on the
        repo.

        """
        attrs = get(self.cfg, self.attrs_iter[0]['_href'])
        counts = attrs.get('content_unit_counts', {})
        self.assertEqual(counts.get('rpm'), 32)
        self.assertEqual(counts.get('erratum'), 4)
        self.assertEqual(counts.get('package_group'), 2)
        self.assertEqual(counts.get('package_category'), 1)
```